braph2genesis/measures/_DegreeOutAv.gen.m

```diff
@@ -0,0 +1,188 @@
+%% ¡header!
+DegreeOutAv < DegreeOut (m, degreeoutav) is the graph degreeoutav.
+
+%%% ¡description!
+The average degreeoutav of a graph is the average of all number of outward edges connected to a node within a layer. 
+
+%% ¡props_update!
+
+%%% ¡prop!
+NAME (constant, string) is the name of the degreeoutav.
+%%%% ¡default!
+'DegreeOutAv'
+
+%%% ¡prop!
+DESCRIPTION (constant, string) is the description of the degreeoutav.
+%%%% ¡default!
+'The average degreeoutav of a graph is the average of all number of outward edges connected to a node within a layer.'
+
+%%% ¡prop!
+TEMPLATE (parameter, item) is the template of the degreeoutav.
+
+%%% ¡prop!
+ID (data, string) is a few-letter code of the degreeoutav.
+%%%% ¡default!
+'DegreeOutAv ID'
+
+%%% ¡prop!
+LABEL (metadata, string) is an extended label of the degreeoutav.
+%%%% ¡default!
+'DegreeOutAv label'
+
+%%% ¡prop!
+NOTES (metadata, string) are some specific notes about the degreeoutav.
+%%%% ¡default!
+'DegreeOutAv notes'
+
+%%% ¡prop!
+SHAPE (constant, scalar) is the measure shape __Measure.GLOBAL__.
+%%%% ¡default!
+Measure.GLOBAL
+
+%%% ¡prop!
+SCOPE (constant, scalar) is the measure scope __Measure.UNILAYER__.
+%%%% ¡default!
+Measure.UNILAYER
+
+%%% ¡prop!
+PARAMETRICITY (constant, scalar) is the parametricity of the measure __Measure.NONPARAMETRIC__.
+%%%% ¡default!
+Measure.NONPARAMETRIC
+
+%%% ¡prop!
+COMPATIBLE_GRAPHS (constant, classlist) is the list of compatible graphs.
+%%%% ¡default!
+{'GraphWD' 'GraphBD' 'MultiplexWD' 'MultiplexBD'}
+
+%%% ¡prop!
+M (result, cell) is the degreeoutav.
```

Comment:
could it be M is the degreeoutav, or M is a cell containing the degree-out value?